service/authorization/authorization.proto

```diff
@@ -7,6 +7,12 @@ import "google/protobuf/any.proto";
 
 import "policy/objects.proto";
 
+
+message Token {
+  string id = 1; // ephemeral id for tracking between request and response
+  string jwt = 2; // the token
+}
```

Comment:
Can we add `TokenCustom`?   like other Custom, `EntityCustom`

Reply:
sure! how would that differ from the string jwt? also what would that look like in terms of selector extraction? we would have to assume the token represented by TokenCustom is a map[string]interface{}